### ISSUE TITLE
Filter bar rendering error when opening a note in the sidebar

### ISSUE THREAD

**Issue by pqnhan** [HEmile/juggl#98]
I just updated Juggle 1.2.1 from Obsidian and got this error.
![Juggle-2022-01-14](https://user-images.githubusercontent.com/24197589/149436850-8892a928-2985-4684-b01c-b931139576ed.gif)

**Comment by HEmile** [comment]
Ah hmm, I think I've seen a similar issue before, where the scroll bar keeps doing weird things, but I could not reproduce it. This shouldn't happen I think if you make the pane Juggl is in slightly larger.

**Comment by pqnhan** [comment]
@HEmile 
I made the Juggle window a bit larger and the filter bar stopped acting weird.
But the horizontal scroll bar below is still the same.
![Peek 2022-01-14 15-24](https://user-images.githubusercontent.com/24197589/149476761-15299b32-0dd8-4fcd-ae29-93683d056efe.gif)

**Comment by HEmile** [comment]
Yes, that's the thing causing this bug. I'm not sure how to fix it though...
